src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -337,6 +337,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\CultureData.Icu.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\CultureData.OSX.cs" Condition="'$(IsOSXLike)' == 'true'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\CultureData.Nls.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\CultureData.WebAssembly.cs" Condition="'$(TargetsBrowser)' == 'true'" />
```

Comment:
-> `CultureData.Browser.cs` would be more specific as this feature doesn't make sense in WASI